.travis/web/app_test.php

```diff
@@ -31,7 +31,7 @@
 Tool::setCurrentRequest($request);
 
 // redirect to installer if pimcore is not installed
-if (!is_file(\Pimcore\Config::locateConfigFile('system.php'))) {
+if (!is_file(\Pimcore\Config::locateConfigFile('system.yml'))) {
```

Comment:
What I described, is to remove this, and add it as import to the file `.travis/app/config/config.yml`. As well for every other occurrence of it